chained_swapper/models/chained_swapper.py

```diff
@@ -89,7 +89,7 @@ def _onchange_model_id(self):
     def write(self, vals):
         super().write(vals)
         if 'name' in vals:
-            self.ref_ir_act_window_id.name = self.name
+            self.filtered("ref_ir_act_window_id").write({"name": vals["name"]})
```

Comment:
Sorry for mistake. It's:

```suggestion
            self.mapped("ref_ir_act_window_id").write({"name": vals["name"]})
```